File: src/main/java/com/indiepost/NewIndiepostApplication.java
package com.indiepost;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.cache.annotation.EnableCaching;


@SpringBootApplication
@EnableConfigurationProperties
@EnableCaching
public class NewIndiepostApplication {

    public static void main(String[] args) {
        SpringApplication.run(NewIndiepostApplication.class, args);
    }
}

File: src/main/kotlin/com/indiepost/NewIndiepostApplication.kt
package com.indiepost

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching


@SpringBootApplication
@EnableConfigurationProperties
@EnableCaching
class NewIndiepostApplication

fun main(args: Array<String>) {
    runApplication<NewIndiepostApplication>(*args)
}
